app/src/components/homePageMainContent/homePageMainContent.tsx

```diff
@@ -28,7 +27,7 @@ const HomePageMainContent = ({
       <DCLane
         key={0}
         records={firstSwimLane.collections}
-        seeMoreLink={`${DC_URL}/collections/lane`}
+        seeMoreLink={`/collections/lane`}
```

Comment:
nit: you don't need braces around strings when setting props. This can be `seeMoreLink="/collections/lane"`.

Reply:
They were needed before since the `DC_URL` var was being interpolated.